apps/ecommerce/frontend/src/helpers/getResourceProviderAndType.ts

```diff
@@ -0,0 +1,12 @@
+import { ExternalResourceLink } from '../types';
```

Comment:
I think the file name should be change to either `utils` or `ResourceProviderUtils`. Something like that